lang/python/python.talon

```diff
@@ -12,6 +12,7 @@ tag(): user.code_functions_common
 tag(): user.code_keywords
 tag(): user.code_libraries
 tag(): user.code_libraries_gui
+tag(): user.code_libraries_gui_showing
```

Comment:
This tag should not be permanently activated. It's used to enable commands that should only be available whilst the relevant window is open.


```suggestion
```